### PR TITLE
Reset password account store

### PR DESCRIPTION
**Pull request by DamirSvrtan**
Support for specifying the account store when reseting passwords. Works both with an href and a nameKey.

### CODE REVIEW

File: lib/stormpath-sdk/resource/application.rb [hunk 1]
```diff
@@ -135,7 +135,9 @@ def api_key_id
     client.data_store.api_key.id
   end
 
-  def create_password_reset_token email
-    password_reset_tokens.create email: email
+  def create_password_reset_token(email, account_store: nil)
```
Comment (ecrisostomo) [inline lib/stormpath-sdk/resource/application.rb]: Not sure if people can send the account store as a resource instance, and not only as a hash, here. Can they?

Comment (DamirSvrtan) [inline lib/stormpath-sdk/resource/application.rb]: @ecrisostomo Yes, you are right, I've pushed a new version where they can send a resource instance.

**Comment by DamirSvrtan** [comment]
@ecrisostomo The build is passing, just take a look if this is okay with you (I believe it should be).

**Comment by ecrisostomo** [comment]
Implementation looks good. Please add the changes to the README.md/CHANGES.md files.

**Pull request merged**